scripts/build_powerplants.py

```diff
@@ -254,7 +254,24 @@ def add_custom_powerplants(ppl, inputs, config):
 
 
 def replace_natural_gas_technology(df):
-    mapping = {"Steam Turbine": "CCGT", "Combustion Engine": "OCGT"}
+    """
+    Maps and replaces gas technologies in the powerplants.csv onto model
+    compliant carriers.
+    """
+    mapping = {
+        "Steam Turbine": "CCGT",
+        "Combustion Engine": "OCGT",
+        "NG": "CCGT",
+        "Ng": "CCGT",
+        "NG/FO": "OCGT",
+        "Ng/Fo": "OCGT",
+        "NG/D": "OCGT",
+        "LNG": "OCGT",
+        "CCGT/D": "CCGT",
+        "CCGT/FO": "CCGT",
+        "LCCGT": "CCGT",
+        "CCGT/Fo": "CCGT",
+    }
```

Comment:
Thanks @Tomkourou for this fix!

I was thinking that it could make sense to parse all the other non-nan but still string entries into the default value ("CCGT").
Currently we fill the nans with CCGT, ok, but I think that:
1. we should check what entries of technologies are not in the set CCGT/OCGT after the fix in line 275
2. when the value is not empty, we shall report their value counts: their number by type with logger
3. those entries shall be replaced to CCGT

What do you think?

Reply:
@Tomkourou Just pointing here to avoid missing notifications.
With too many tags, sometimes I get lost...

Reply:
> Thanks @Tomkourou for this fix!
> 
> I was thinking that it could make sense to parse all the other non-nan but still string entries into the default value ("CCGT"). Currently we fill the nans with CCGT, ok, but I think that:
> 
> 1. we should check what entries of technologies are not in the set CCGT/OCGT after the fix in line 275
> 2. when the value is not empty, we shall report their value counts: their number by type with logger
> 3. those entries shall be replaced to CCGT
> 
> What do you think?

Just to avoid misunderstanding, it seems this task remained pending, not sure if you were planning on handling it.